lib/osf-components/addon/components/cedar-metadata-editor/component.ts

```diff
@@ -63,9 +82,11 @@ export default class CedarMetadataEditor extends Component<Args> {
             record.target = this.args.target;
         }
 
+        record.isPublished = this.isValid;
```

Comment:
Is this `isPublished` flag something we want to be setting on the FE? This seems like a field that should be updated on the BE